libp2p/protocols/mix/entry_connection.nim

```diff
@@ -0,0 +1,144 @@
+import hashes, chronos, stew/byteutils, results, chronicles
+import ../../stream/connection
+import ../../varint
+import ../../utils/sequninit
+import ./mix_protocol
+from fragmentation import DataSize
+
+type MixDialer* = proc(
+  msg: seq[byte], codec: string, destination: MixDestination
+): Future[void] {.async: (raises: [CancelledError, LPStreamError], raw: true).}
+
+type MixEntryConnection* = ref object of Connection
+  destination: MixDestination
+  codec: string
+  mixDialer: MixDialer
+
+func shortLog*(conn: MixEntryConnection): string =
+  if conn == nil:
+    "MixEntryConnection(nil)"
+  else:
+    "MixEntryConnection(" & $conn.destination & ")"
+
+chronicles.formatIt(MixEntryConnection):
+  shortLog(it)
+
+method readOnce*(
+    s: MixEntryConnection, pbytes: pointer, nbytes: int
+): Future[int] {.async: (raises: [CancelledError, LPStreamError]), public.} =
+  # TODO: implement
```

Comment:
raiseAssert "not immplemented"

Reply:
Ah! these were immediatly implemented in https://github.com/vacp2p/nim-libp2p/blob/3a5d2ea4d7270df31c63622030841b6b25f84912/libp2p/protocols/mix/entry_connection.nim#L39-L161 
Will keep it like this for this PR since they're gonna be removed immediatly anyway